entity-service/src/main/kotlin/com/egm/stellio/entity/repository/Neo4jRepository.kt

```diff
@@ -250,7 +251,7 @@ class Neo4jRepository(
                         it.third.isTime() -> "localtime('${it.third}')"
                         else -> "'${it.third}'"
                     }
-                    "(n)-[:HAS_VALUE]->(${it.first.extractShortTypeFromExpanded()}: Property { name: '${it.first}'}) and ${it.first.extractShortTypeFromExpanded()}.value ${it.second} $comparableValue"
+                    "(n)-[:HAS_VALUE]->(${it.first.extractShortTypeFromExpanded()}:Property { name: '${it.first}'}) and ${it.first.extractShortTypeFromExpanded()}.value ${it.second} $comparableValue"

```

Comment:
This line is way too long

Reply:
Well, it was also before but I try to avoid mixing code formatting and feature / fix in a PR because that makes code reviews painful.